project/Vagrantfile

```diff
@@ -65,13 +69,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         master.vm.box = "chef/centos-6.5"
         master.vm.provider "virtualbox" do |v|
             v.name = mastername
-            v.memory = 2048
-            v.cpus = 2
+            v.memory = @memory
+            v.cpus = @cpus
         end
         master.vm.hostname = mastername
         master.vm.network "public_network", bridge: 'en0: Wi-Fi (AirPort)'
+        #master.vm.network "private_network", type: "dhcp"
         master.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
         master.vm.synced_folder "/space/software", "/space/software"
+        master.vm.synced_folder Dir.getwd, "/project/root"
```

Comment:
This is not really necessary. The current dir is automatically mounted as /vagrant..